.changeset/green-lions-mix.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Changed: `declaration-block-no-duplicate-properties` warning ranges are now always matching the duplicate that will be removed after autofixing
```

Comment:
[suggest] "Changed" has often been used for significant changes, so I'd like to use "Fixed" here. So, how about this?

```suggestion
Fixed: `declaration-block-no-duplicate-properties` reported code ranges
```

Reply:
```suggestion
Fixed: `declaration-block-no-duplicate-properties` reported ranges
```